README.adoc

```diff
@@ -1,4 +1,4 @@
-= Cucumber based blackbox testing framework for OpenShift
+= Cucumber based blackbox testing framework for OpenShift and Kata tests
```

Comment:
@valtair1891 I don't think we should be adding `and Kata tests`.  Cucushift is used for testing OCP and many other operators, so no need to list out `kata` specificially.